services/src/test/java/org/jfrog/artifactory/client/aql/AqlQueryBuilderTest.java

```diff
@@ -0,0 +1,252 @@
+package org.jfrog.artifactory.client.aql;
```

Comment:
This test class includes the following formatter comments multiple times:
//@formatter:off
//@formatter:on
Let's remove them.

Reply:
removeed